src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected static int numTask = 0;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        numTask++;
+    }
+
+    public static int getNumTask() {
```

Comment:
Suggestion: Same with the previous comment, this method could be renamed as "getTaskNum"